search/src/components/BookmarkPopover.tsx

```diff
@@ -31,7 +31,7 @@ export interface BookmarkPopoverProps {
 }
 
 const BookmarkPopover = (props: BookmarkPopoverProps) => {
-  const apiHost = import.meta.env.VITE_API_HOST as string;
+const apiHost = window.API_HOST || import.meta.env.VITE_API_HOST as string;
```

Comment:
Fwiw, my PR literally does the samr thing, but it uses a VITE plug-in to do this instead of hard coding.